src/ECPublicKeyTrait.php

```diff
@@ -10,12 +10,23 @@
     // Stored base64-encoded
     private $pubKey = '';
 
-    // Binary string of public key
+    /**
+     * @deprecated Methods that return binary values are suffixed with "binary".
+     * @return string Binary string of public key
+     */
     public function getPublicKey(): string
     {
         return base64_decode($this->pubKey);
     }
 
+    /**
+     * @return string The decoded public key.
+     */
+    public function getPublicKeyBinary(): string
```

Comment:
I like the consistency that this naming change adds. It may make sense to do the same with `setPublicKey` in the future, but let's keep that out of the discussion for now.

Would you mind updating the couple of usages in `Server` (and this trait's corresponding unit test) to use the new method as well?